app/views/signs/show.html.haml

```diff
@@ -17,6 +17,10 @@
         %h2.main_gloss= @sign.gloss_main
         %h2.main_gloss.maori-gloss= @sign.gloss_maori
       .buttons-container
+        = link_to "/images/signs/400-400/#{convert_to_high_resolution(@sign.drawing)}", { class: "download-link", "data-sign-id": @sign.id } do
```

Comment:
questions -- is this a cache of images from Freelex server? What if the file isn't there yet? What if the file doesn't exist?